k8s/configuration/secret.yaml

```diff
@@ -0,0 +1,92 @@
+# All secret data should be base64 encoded before embedding them here by
+# using `echo "secret string" | base64 -w 0 > secret.string.b64` and then
+# copy the resulting value here.
+# Ref: https://kubernetes.io/docs/concepts/configuration/secret/
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-agent-api-key
+  namespace: default
+type: Opaque
+data:
+  # This is the API Key obtained from MongoDB Cloud Manager
+  api-key: "<b64 encoded api key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bdb-private-key
+  namespace: default
+type: Opaque
+data:
+  private.key: "<b64 encoded private key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-certs
+  namespace: default
+type: Opaque
+data:
+  mdb-instance.pem: "<b64 encoded, concatanated public and private keys>"
+  ca.pem: "<b64 encoded CA public key>" 
```

Comment:
A `ca.pem` data item appears in several places in `secret.yaml`, but I think a given BigchainDB Cluster will have only one CA, and therefore only one `ca.pem`. Maybe there could be a separate Kubernetes Secret named `ca` with the `ca.pem` in it?